### PR TITLE
Fixed the stdout redirect in launch_ccm_cluster.py

### PR DESCRIPTION
**Pull request by susanxhuynh**
Made sure the stdout redirect is applied in the subprocess call to create_service_account.sh. Previously, the output from the shell script was still appearing in stdout.

### CODE REVIEW

File: tools/launch_ccm_cluster.py [hunk 1]
```diff
@@ -287,7 +287,7 @@ def run_script(scriptname, args = []):
                 stdout = sys.stdout
                 sys.stdout = sys.stderr
                 script_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), scriptname)
-                subprocess.check_call(['bash', script_path] + args)
+                subprocess.check_call(['bash', script_path] + args, stdout=sys.stdout)
```
Comment (nickbp) [inline tools/launch_ccm_cluster.py]: Could the surrounding redirect be removed in favor of setting `stdout=sys.stderr` here?

**Comment by susanxhuynh** [comment]
Removed the surrounding stdout redirect.

**Comment by susanxhuynh** [comment]
retest this please

**Comment by susanxhuynh** [comment]
retest this please

**Comment by susanxhuynh** [comment]
retest this please

**Comment by susanxhuynh** [comment]
retest this please

**Pull request merged**